client/src/components/DropdownMenu.scss

```diff
@@ -3,6 +3,8 @@
 .dropdown {
```

Comment:
Hmm, these class names didn't come from your commits, but I'm going to comment on them while I'm here.

`dropdown__content__is-open` should be `dropdown__content--is-open`. Better yet though, I think `--is-open` should modify `dropdown` itself.

And all of
```
<button
  ...
>
  {isOpen ? (
    <div className="close-dropdown">
      <span
        aria-label={trivial_text_maker("close")}
        className="close-dropdown__x"
      >
        X
      </span>
      <span className="close-dropdown__x" aria-label={aria_label}>
        {dropdown_trigger_txt}
      </span>
    </div>
  ) : (
    <span aria-label={`${trivial_text_maker("open")} ${aria_label}`}>
      {dropdown_trigger_txt}
    </span>
  )}
</button>
```

Could be 
```
<button
  ...
>
  {dropdown_trigger_txt}
</button>
```
with:
  - `aria-expanded` replacing the closed/open in the label
  - this scss doing everything else:
```
.dropdown--is-open {
  & > button {
   &::before {
      content: "X";
      color: $highlightColor !important;
    }
    &:hover, &:focus {
      color: $tertiaryColor;
    }
  }

  & > .dropdown__content {
    display: block;
  }
}
```

Reply:
The `!important` on the X's colour's because of the hover over-ridding it, right? I guess I can't think of a reason for that X to not be red later, so it's fine, but still better generally to avoid `!important`. Almost never the only option.

Reply:
seems like the `!important` isn't needed!